src/views/common/Animation/AnimationControls/PlaybackControlButtonLogic.ts

```diff
@@ -0,0 +1,80 @@
+import React, { useCallback, useMemo } from "react"
+import { AnimationStateDispatcher } from "../AnimationStateReducer"
+
+export type PlaybackButtonLogicCallback = (e: React.MouseEvent<any> | React.KeyboardEvent<HTMLDivElement>) => void
+export type KeyHandler = (e: React.KeyboardEvent<HTMLDivElement>) => void
+
+export type PlaybackControlLogic = {
+    returnToBeginningHandler: PlaybackButtonLogicCallback
+    backSkipHandler: PlaybackButtonLogicCallback
+    forwardSkipHandler: PlaybackButtonLogicCallback
+    jumpToEndHandler: PlaybackButtonLogicCallback
+    playPauseHandler: PlaybackButtonLogicCallback
+    reverseDirectionHandler: PlaybackButtonLogicCallback
+    syncWindowHandler: PlaybackButtonLogicCallback
+    cropWindowHandler: PlaybackButtonLogicCallback
+    keyboardControlHandler: KeyHandler
+}
```

Comment:
This mainly serves as a manifest for the types of control logic exposed here.

It may have been desirable to bring this closer to the `AnimationStateReducer`--that's worth considering in the future.